### PR TITLE
Issue #SB-1402 chore: Portal document update

### PR DESCRIPTION
**Pull request by harishgilimi**
https://github.com/project-sunbird/project-sunbird.github.io/issues/24#issuecomment-357860514

### CODE REVIEW

**Comment by lakhanmandloi** [comment]
Hello @harishgilimi,<br/> _This is a auto-generated response._ <br/> Click the URL for review pull request - http://sunbird-docs-qa.s3-website.ap-south-1.amazonaws.com/pr/212.

**Comment by harishgilimi** [comment]
@deepshikha2k5  please suggest if any changes required

cc: @kochhar

File: pages/developer-docs/installation/installing_sunbirdon_laptop.md [hunk 1]
```diff
@@ -57,6 +57,8 @@ After checking out the code, execute the following command:
 
     git clone https://github.com/project-sunbird/sunbird-portal.git
 
+Note: The sunbird portal stable versions are available in tags for each release and master branch contains the latest stable release.
```
Comment (kochhar) [inline pages/developer-docs/installation/installing_sunbirdon_laptop.md]: @harishgilimi please add a link or instructions for getting a stable release from a tag. We cannot assume that the reader will know how to get the release from a tag.

**Comment by nawabahmadreshi** [comment]
@harishgilimi, should I merge the PR or not, since you have not replied to @kochhar comment.

File: pages/developer-docs/installation/installing_sunbirdon_laptop.md (hunk 1 above)
Comment (harishgilimi) [reply pages/developer-docs/installation/installing_sunbirdon_laptop.md]: @kochhar  sure i will add it

**Comment by harishgilimi** [comment]
@nawabahmadreshi  please don't merge i will update the PR

**Comment by nawabahmadreshi** [comment]
@harishgilimi ok, awaiting your green signal.

**Comment by harishgilimi** [comment]
@nawabahmadreshi sorry for late reply updated PR please check

**Comment by harishgilimi** [comment]
https://github.com/project-sunbird/project-sunbird.github.io/pull/248

**Pull request closed without merging**